src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started working.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stop working.");
```

Comment:
There's a typo in the message: 'Excavator stop working.' should be 'Excavator stopped working.' to maintain consistency with the past tense used in the `doWork` method.